app/models/labware_creators/plate_split_to_tube_racks.rb

```diff
@@ -218,23 +217,24 @@ def extract_barcodes(file)
       file.position_details.values.pluck('tube_barcode')
     end
 
-    # Validation that checks if there are sufficient tubes in the child tube racks for all the parent wells.
+    # Validation that checks if there are the correct number of tubes in the child tube racks for all
+    # the parent wells.
     # This depends on the number of unique samples in the parent plate, and the number of parent wells,
     # as well as whether they are using both sequencing tubes and contingency tubes or just contingency.
     #
-    # Sets errors if there are insufficient tubes.
-    def must_have_sufficient_tubes_in_rack_files
+    # Sets errors if there are insufficient or too many tubes.
+    def must_have_correct_number_of_tubes_in_rack_files
       return unless files_valid?
 
       if require_contingency_tubes_only?
-        add_error_if_insufficient_tubes(:contingency_csv_file, num_contingency_tubes, num_parent_wells)
+        add_error_if_wrong_number_of_tubes(:contingency_csv_file, num_contingency_tubes, num_parent_wells)
       else
-        add_error_if_insufficient_tubes(
+        add_error_if_wrong_number_of_tubes(
           :contingency_csv_file,
           num_contingency_tubes,
           num_parent_wells - num_parent_unique_samples
```

Comment:
We talked about if `num_parent_wells - num_parent_unique_samples` is zero, then `required_tubes` argument will be zero, rejecting any number of contingency tubes.